src/components/ProgressBar/ProgressBar.tsx

```diff
@@ -0,0 +1,76 @@
+import React, { useEffect, useState } from 'react';
+
+import style from './ProgressBar.module.css';
+
+type Props = {
+  percentage: number;
+  textPosition: 'in' | 'up';
+  textAdded?: string;
+};
+
+function getColor(percentage: number) {
+  const red = '#D76B65';
+  const blue = '#2D689F';
+  const green = '#37A169';
+
+  if (percentage < 70) return red;
+  if (percentage < 90) return blue;
+  return green;
+}
+
+export default function ProgressBar({
+  percentage,
+  textPosition,
+  textAdded,
+}: Props) {
+  const text = ` ${textAdded}`;
+  const [progress, setProgress] = useState(0);
+  useEffect(() => {
+    setTimeout(() => {
+      if (progress < percentage) {
+        setProgress(progress + 1);
```

Comment:
I don't think we should be using Timeouts for loading animations. But if there's no other way we can keep it that way.